components/Cards/card.css

```diff
@@ -0,0 +1,231 @@
+.content-container {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    padding: 0.5rem;
+}
+
+.content-container h1 {
+    font-size: 1.5rem;
+    font-weight: 500;
+}
+
+.content-container h3 {
+    font-size: 1rem;
+    color: var(--dark-gray);
+    font-weight: 400;
+}
+
+.content-container p {
+    padding-top: 1rem;
+    color: var(--light-gray);
+    font-size: 0.9rem;
+}
+
+.action-items-container {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    width: fit-content;
+}
+
+.btn-container {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: flex-start;
+    width: fit-content;
+}
+
+.btn {
+    border: none;
+    background-color: transparent;
+    color: var(--primary-color);
+    text-transform: uppercase;
+    cursor: pointer;
+    font-size: 1rem;
+    border-radius: 10%;
+    -webkit-border-radius: 10%;
+    -moz-border-radius: 10%;
+    -ms-border-radius: 10%;
+    -o-border-radius: 10%;
+    width: fit-content;
+    margin:0.5rem;
+    padding: 0.5rem;
+}
+
+.btn:hover, i:hover {
+    box-shadow: -1px 1px 1px rgba(0,0,0,0.5)
+}
+
+.icon-container {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    font-weight: 100;
+    text-align: center;
+    width: fit-content;
+}
+
+.icon-container>* {
+    padding-top: 0.7rem;
+}
+
+i {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+    font-size: 1.3rem;
+    height: 2.6rem;
+    width: 2.6rem;
+    border-radius: 50%;
+    -webkit-border-radius: 50%;
+    -moz-border-radius: 50%;
+    -ms-border-radius: 50%;
+    -o-border-radius: 50%;
+}
+
+
+
+/* Variants */
+
+.card-text-over-media {
+    position: relative;
+    height: 24rem;
+}
+
+.card-text-over-media>.content-container>h1 {
+    top: 10rem;
+
+}
+
+.card-text-over-media>.content-container>h3 {
+    top: 12.5rem;
+}
+
+.card-text-over-media>.content-container>h1,
+.card-text-over-media>.content-container>h3 {
+    position: absolute;
+    color: var(--white);
+}
+
+.card-text-over-media>.content-container>p {
+    padding-top: 0.5rem;
+}
+
+/* Basic Card with Header */
+
+.card-basic-with-header {
+    height: 29rem;
+}
+
+.card-basic-with-header>.content-container>p {
+    padding-top: 0.2rem;
+}
+
+.card-basic-with-header>.image-container>img {
+    border-radius: 0rem;
+    -webkit-border-radius: 0rem;
+    -moz-border-radius: 0rem;
+    -ms-border-radius: 0rem;
+    -o-border-radius: 0rem;
+}
+
+/* Card with only buttons */
+
+.card-with-buttons>.action-items-container>.icon-container,
+.card-with-icons>.action-items-container>.btn-container {
+    display: none;
+}
+
+/* Card with only icons */
+.card-with-icons>.action-items-container>.icon-container {
+    padding-top: 1rem;
+    padding-left: 12rem;
+
+}
+
+/* Horizontal card with image and text */
+
+.card-horizontal , .card-horizontal img {
```

Comment:
Here you could give the image a `border-bottom-left-radius` and remove the(or set to initial) `border-top-right-radius` which makes the card look better.

![Screenshot 2022-01-31 at 00 37 37](https://user-images.githubusercontent.com/62737267/151713845-7d0fd6f4-e410-41e3-8989-51ad72277d53.png)

Reply:
Great efforts vishal.

Reply:
Thanks for the suggestion. I added it and it surely looks better.